manifests/o/Oracle/MySQL/8.0.26.0/Oracle.MySQL.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using YamlCreate.ps1 v1.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Oracle.MySQL
+PackageVersion: 8.0.26.0
```

```suggestion
PackageVersion: 1.4.38.0
```
You'll have to update other MySQL manifests as well since in Control Panel `{9E43FA92-14E1-4CA3-8DAA-39F19EEF6E65}` shows up as `1.4.38.0` rather than `8.0.26.0`.